python/ray/autoscaler/_private/commands.py

```diff
@@ -906,28 +916,29 @@ def exec_cluster(config_file: str,
         provider.cleanup()
 
 
-def _exec(updater,
-          cmd,
-          screen,
-          tmux,
-          port_forward=None,
-          with_output=False,
-          run_env="auto",
-          shutdown_after_run=False):
+def _exec(updater: NodeUpdaterThread,
+          cmd: Optional[str] = None,
+          screen: bool = False,
+          tmux: bool = False,
+          port_forward: Union[Tuple[int, int], List[Tuple[int, int]],
+                              None] = None,
+          with_output: bool = False,
+          run_env: str = "auto",
+          shutdown_after_run: bool = False) -> str:
     if cmd:
         if screen:
-            cmd = [
+            wrapped_cmd = [
```

Comment:
Oh that is much better!

Reply:
It was just to appease mypy's complaint about the type mismatch.

Reply:
ahahaha, well it looks cleaner!